package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "@stylelint/prettier-config",
+  "version": "1.0.0",
+  "description": "stylelint org's shareable config for prettier",
+  "keywords": [
+    "prettier",
+    "prettierconfig"
+  ],
+  "repository": "stylelint/prettier-config",
+  "license": "MIT",
+  "author": "stylelint",
+  "main": "index.js",
+  "files": [
+    ".prettierrc.js"
+  ],
+  "scripts": {
+    "format": "prettier . --write",
+    "lint": "npm-run-all --parallel lint:*",
+    "lint:formatting": "prettier . --check --ignore-path .gitignore",
+    "lint:js": "eslint . --ignore-path .gitignore",
+    "release": "np"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
+  "lint-staged": {
+    "*.js": "eslint --cache --fix",
+    "*.{js,json,md,yml}": "prettier --write"
+  },
+  "eslintConfig": {
+    "extends": [
+      "stylelint"
+    ]
+  },
+  "dependencies": {
+    "prettier-plugin-packagejson": "^2.1.2"
```

Comment:
Can we have it explicitly enabled in prettierrc, please? I wouldn't know about this plugin running with Prettier, if didn't see it in list of dependencies.

Reply:
I'm not sure it's possible in the configuration object.

[Accordingly to the docs](https://prettier.io/docs/en/plugins.html#using-plugins), it can only be done using CLI flags or API arguments.

---

I'm keen to use the plugin as we use `package.json` for both run-scripts and their configurations, and each repo currently has them in different orders making it confusing moving between repos.